client/fingerprint/cpu_default.go

```diff
@@ -3,6 +3,6 @@
 
 package fingerprint
 
-func (f *CPUFingerprint) deriveReservableCores(req *FingerprintRequest) ([]uint16, error) {
-	return nil, nil
+func (f *CPUFingerprint) deriveReservableCores(string) []uint16 {
```

Comment:
nit: ignore the unused parameter
```suggestion
func (f *CPUFingerprint) deriveReservableCores(_ string) []uint16 {
```